### DISCUSSION TITLE
Sections for instructions

### DISCUSSION THREAD

**Discussion by simgunz** [mealie-recipes/mealie#1181] [Feature Request]
Some recipes contains instructions divided in sections. The parser seems to parse the sections correctly but then all the steps are merged togheter in the mealie interface. Would be good to have the option to create instructions sections.

Example recipe:
https://redhousespice.com/red-bean-paste/

**Comment by ZacWarham** [comment]
Has this been road mapped yet?

**Comment by michael-genson** [comment]
While the parser doesn't do this, you can add ingredient sections and instruction sections already in Mealie by clicking the three dots on any instruction/ingredient

![image](https://github.com/mealie-recipes/mealie/assets/71845777/753ccdcf-63f8-4b99-be88-d56b2a3c8ee6)

![image](https://github.com/mealie-recipes/mealie/assets/71845777/818863a9-7e52-4c79-b2e7-64eee4822166)

**Reply by ZacWarham** [reply]
That seems sufficient enough for my purposes :) thanks for pointing that out!

Although I expected toggle section would convert the step into a section, using the text as heading or create a new step as opposed to just a new heading but I guess it was done this way works as it would likely be the best for most recipe types when importing.

**Reply by boomshadow** [reply]
@michael-genson Thank you for pointing that out! I came here to feature request sections for ingredients and instructions, not realizing this was already supported. It wasn't obvious to me. Your screenshots are great! Thank you! This is exactly what I was looking for in the feature 😁

By the way, thank you so much for the OpenAI feature work you've been doing. I already love Mealie, but the integration has made it sooooo much better! Thank you, thank you!
